src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
         {
             foreach (ParameterSymbol param in property.Parameters)
             {
-                if (param.RefKind != RefKind.None)
+                if (param.RefKind == RefKind.Ref || param.RefKind == RefKind.Out)
```

Comment:
> Adding `IsConstModReqIsConsumedInRefCustomModifiersPosition_Indexers_ReturnTypes`. This change basically allows to pass ref-readonly arguments to indexer methods. Passing ordinary arguments is the same to properties as ref-readonly arguments.

I couldn't find the test, what file it should be in?

---
In reply to: [118609191](https://github.com/dotnet/roslyn/pull/19658#discussion_r118609191) [](ancestors = 118609191,118061963)